doc/symfony-cache-configuration.rst

```diff
@@ -129,6 +129,29 @@ other hosts, provide the IPs of the machines allowed to purge, or provide a
 RequestMatcher that checks for an Authorization header or similar. *Only set
 one of ``client_ips`` or ``client_matcher``*.
 
+Symfony's `HttpCache` does not support cache invalidation by tags by default.
+However, this library ships with a `TaggableStore` that provides exactly that.
+If you want to use this, adjust your `AppCache` as follows::
+
+    use FOS\HttpCache\SymfonyCache\TaggableStore();
+
+    // ...
```

Comment:
please have the `class AppCache extends HttpCache` line here

Reply:
Same here, I guess the people know about `app/AppCache.php` but I added a hint in 0e3d337e5e1e43cfc802d2ef74997c82432c5c38 anyway (for the first example to not mess up with all other examples).